### PR TITLE
[MODROLESKC-211] Implement api to update roleCapabilities and roleCapabilitySets by name

### PR DESCRIPTION
**Pull request by Saba-Zedginidze-EPAM**
## Purpose
[[MODROLESKC-211] Implement api to update roleCapabilities and roleCapabilitySets by name](https://folio-org.atlassian.net/browse/MODROLESKC-211)

## Approach
- Refactor update endpoint payloads to accept both capability and capabilitySet ids and names
- Update unit and integraton tests

### CODE REVIEW

File: src/main/java/org/folio/roles/service/capability/RoleCapabilityServiceImpl.java [hunk 1]
```diff
@@ -115,6 +117,22 @@ public void update(UUID roleId, List<UUID> capabilityIds) {
...
+   *
+   * @param roleId - role identifier as {@link UUID} object
+   * @param request - CapabilitiesUpdateRequest that contains either capability IDs or names, to be assigned to a role
+   */
+  @Override
+  @Transactional
+  public void update(UUID roleId, CapabilitiesUpdateRequest request) {
+    verifyRequest(request);
+    var resolvedCapabilitiesIds = resolveCapabilitiesByNames(request.getCapabilityNames());
+    var allCapabilityIds = CollectionUtils.union(resolvedCapabilitiesIds, request.getCapabilityIds());
```
Comment (BKadirkhodjaev) [inline src/main/java/org/folio/roles/service/capability/RoleCapabilityServiceImpl.java:132]: An interesting way to support both cases

File: src/main/resources/swagger.api/schemas/capability/capabilitiesUpdateRequest.json [hunk 2]
```diff
@@ -13,7 +12,14 @@
         "description": "Capability identifier",
         "format": "uuid"
       }
+    },
+    "capabilityNames": {
```
Comment (BKadirkhodjaev) [inline src/main/resources/swagger.api/schemas/capability/capabilitiesUpdateRequest.json:16]: Is there any max count, say 100 capability names per request or it is unbound?

Comment (Saba-Zedginidze-EPAM) [reply src/main/resources/swagger.api/schemas/capability/capabilitiesUpdateRequest.json:16]: There is not bound for it

**Review by BKadirkhodjaev — APPROVED** [review]

**Review by SerhiiNosko — APPROVED** [review]

**Review by azizbekxm — APPROVED** [review]

**Comment by SerhiiNosko** [comment]
@folio-org/eureka can you take a look on this PR please
It is needed for 5 UI stories that are in the current sprint:
https://folio-org.atlassian.net/browse/UICONSET-173
https://folio-org.atlassian.net/browse/UICONSET-177
https://folio-org.atlassian.net/browse/UISAUTHCOM-14
https://folio-org.atlassian.net/browse/UIROLES-95
https://folio-org.atlassian.net/browse/UIPOLICIES-12

**Pull request merged**